tests/test_diffeq/test_perturbedsolvers/test_perturbation_functions.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+
+from probnum.diffeq.perturbedsolvers import _perturbation_functions
+
+random_state = np.random.mtrand.RandomState(seed=1)
+
+
+@pytest.fixture
+def step():
+    return 0.2
+
+
+@pytest.fixture
+def solver_order():
+    return 4
+
+
+@pytest.fixture
+def noise_scale():
+    return 1
+
+
+@pytest.fixture
+def num_samples():
+    return 100
+
+
+def test_perturb_uniform_mean(step, solver_order, noise_scale, num_samples):
+    suggested_steps = _perturbation_functions.perturb_uniform(
+        step, solver_order, noise_scale, random_state=1, size=num_samples
+    )
+    mean_suggested_step = np.sum(suggested_steps) / num_samples
+    np.testing.assert_allclose(mean_suggested_step, step, atol=1e-4, rtol=1e-4)
+
+
+def test_perturb_lognormal_mean(step, solver_order, noise_scale, num_samples):
+    suggested_steps = _perturbation_functions.perturb_lognormal(
+        step, solver_order, noise_scale, random_state=1, size=num_samples
+    )
+    mean_suggested_step = np.sum(suggested_steps) / num_samples
+    np.testing.assert_allclose(mean_suggested_step, step, atol=1e-6, rtol=1e-6)
```

Comment:
these tests are identical, so you could write the test once (and make the `perturb_x` functions into inputs that are then parameters of the test). This would make it easier to run the tests for more perturbation functions :)